flytekit/annotated/interface.py

```diff
@@ -1,7 +1,7 @@
 import copy
 import inspect
 from collections import OrderedDict
-from typing import Dict, Generator, Union, Type, Tuple, List, Any, _GenericAlias, TypeVar
```

Comment:
_GenericAlias gives problems in cookbook, I think cuz the cookbook image uses 3.6.  I don't think we should support 3.6, but I also don't like the idea of importing _GenericAlias... Take a look at the change below to see if it's acceptable.